stf.c

```diff
@@ -53,6 +59,13 @@ void setPolar(float a, float b, float c, float w){
   updateRealPolar();
 } 
 
+void setRealPolar(float a, float b, float c){
+  polar.a=a;
```

Comment:
Let's keep indentation consistent with the rest of the source code: tabs, no spaces.

Reply:
I'll fix it when I get there.